Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -1469,120 +3602,842 @@ def get_task_status_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict
+    Creates an Intrusion Policy with the specified parameters.
+
+    Args:
+        client (Client): Session to Cisco Firepower Management Center to run desired requests.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about the created Intrusion Policy
+    """
+    name = args['name']
+    basepolicy_id = args['basepolicy_id']
+    description = args.get('description')
+    inspection_mode = args.get('inspection_mode')
+
+    raw_response = client.create_intrusion_policy(
+        name=name,
+        basepolicy_id=basepolicy_id,
+        description=description,
+        inspection_mode=inspection_mode,
+    )
+
+    return get_commandresults(
+        raw_response=raw_response,
+        command_headers_by_keys=INTRUSION_POLICY_HEADERS_BY_KEYS,
+        command_title=f'Created {INTRUSION_POLICY_TITLE}',
+        command_context=INTRUSION_POLICY_CONTEXT,
+    )
+
+
+def list_intrusion_policy_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Retrieves the intrusion policy associated with the specified ID.
+    If no ID is specified, retrieves list of intrusion policies.
+    - GET arguments: intrusion_policy_id, include_count.
+    - LIST arguments: expanded_response, limit, page, page_size.
+
+    Args:
+        client (Client): Session to Cisco Firepower Management Center to run desired requests.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about Intrusion Policies.
+    """
+    # GET arguments
+    intrusion_policy_id = args.get('intrusion_policy_id', '')
+    include_count = argToBoolean(args.get('include_count', 'False'))
+    # LIST arguments
+    limit = arg_to_number(args.get('limit', 0))
+    page = arg_to_number(args.get('page', 0))
+    page_size = arg_to_number(args.get('page_size', 0))
+    expanded_response = argToBoolean(args.get('expanded_response', 'False'))
+
+    is_get_request = check_is_get_request(
+        get_args=[intrusion_policy_id, include_count],
+        list_args=[limit, page, page_size, expanded_response]
+    )
+
+    if is_get_request:
+        raw_response = client.get_intrusion_policy(
+            intrusion_policy_id=intrusion_policy_id,
+            include_count=include_count,
+        )
+
+    else:  # is_list_request
+        raw_response, raw_responses = client.list_intrusion_policy(
+            page=page,
+            page_size=page_size,
+            limit=limit,
+            expanded_response=expanded_response,
+        )
+
+    command_results = get_commandresults(
+        raw_response=raw_response,
+        command_headers_by_keys=INTRUSION_POLICY_HEADERS_BY_KEYS,
+        command_title=f'Fetched {INTRUSION_POLICY_TITLE}',
+        command_context=INTRUSION_POLICY_CONTEXT,
+    )
+
+    if not is_get_request:
+        command_results.raw_response = raw_responses
```

Comment:
Because of `pagination` we may have to make several API requests to the server, for example: max `limit=10` and the user wanted `limit=30` which results in 3 `raw_response`. How should I handle those 3 `raw_response` with the actual `items` that are supposed to be returned to the user.

What should I do?
- Edit the 1st `raw_response` to hold the rest (not too fond of this).
- Create a `CommandResults` for each `raw_response` that will also hold the related `context_outputs` (this will result in a `readable_output` that is split to multiple pieces).
-  Leave it as is.
- Return all the `context_outputs` and `readable_output` (`items`)  in the first or last `CommandResults` and the rest of the `CommandResults` will hold `raw_responses` (this will result in 1 full `readable_output` and empty `readable_outputs` that will be sent).

Reply:
Understood, let's leave it as it is, thanks